build.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -e
+
+BUILD_TARGETS="darwin/amd64 linux/386 linux/amd64 windows/386 windows/amd64"
+
+FOLDERS_TO_BUILD=(
```

Comment:
Do we just want to build every examples directory, rather than having to specifically add it here?